src/Shared/ServerInfrastructure/BufferExtensions.cs

```diff
@@ -26,6 +26,27 @@ public static ReadOnlySpan<byte> ToSpan(in this ReadOnlySequence<byte> buffer)
             return buffer.ToArray();
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static void CopyTo(in this ReadOnlySequence<byte> buffer, PipeWriter pipeWriter)
```

Comment:
Maybe we should add this to ReadOnlySequence the target would be an IBufferWriter

Reply:
Add it to runtime? Could do. It has this today:

```cs
    public static class BuffersExtensions
    {
        public static void Write<T>(this IBufferWriter<T> writer, ReadOnlySpan<T> value)
        {
        }
    }
```

Reply:
https://github.com/dotnet/runtime/issues/25443